Add regex filter to audit

I have a large number of legacy passwords that share a common formula that I'd like to clean up and bring able to use a regex during audit would help me track all of these passwords down.  I think this might be a fairly common problem for a lot of people who are just getting started using a password manager and are importing all of their legacy account accesses.

That works perfectly!  I'm currently running the following version
```
gopass 1.10.1-git+HEAD go1.13.15 linux amd64
```
Is the `grep` command added in the help pages in the current version? Unexpectedly, I can't seem to get `v1.12.1` to build due to several `os` level packages that are claimed to not exist.

For example
```
# github.com/gopasspw/gopass/pkg/tempfile
pkg/tempfile/file.go:24:13: undefined: os.MkdirTemp
# github.com/gopasspw/gopass/internal/cache
internal/cache/disk.go:47:14: undefined: os.ReadFile
internal/cache/disk.go:57:9: undefined: os.WriteFile
# github.com/gopasspw/gopass/pkg/gopass/secrets/secparse
pkg/gopass/secrets/secparse/mime.go:45:33: hdr.Values undefined (type textproto.MIMEHeader has no field or method Values)
# github.com/gopasspw/gopass/internal/config
internal/config/io.go:69:14: undefined: os.ReadFile
internal/config/io.go:191:12: undefined: os.WriteFile
# github.com/gopasspw/gopass/pkg/otp
pkg/otp/otp.go:65:12: undefined: os.WriteFile
# github.com/gopasspw/gopass/internal/backend/storage/fs
internal/backend/storage/fs/fsck.go:141:13: undefined: os.ReadDir
internal/backend/storage/fs/store.go:42:9: undefined: os.ReadFile
internal/backend/storage/fs/store.go:58:9: undefined: os.WriteFile
# github.com/gopasspw/gopass/pkg/clipboard
pkg/clipboard/clipboard_others.go:42:18: undefined: os.ReadFile
```

--- a/internal/action/commands.go
+++ b/internal/action/commands.go
@@ -509,7 +509,6 @@ func (s *Action) GetCommands() []*cli.Command {
 				"content.",
 			Before: s.IsInitialized,
 			Action: s.Grep,
-			Hidden: true,
 			Flags: []cli.Flag{
 				&cli.BoolFlag{
 					Name:    "regexp",

